src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs

```diff
@@ -547,10 +545,15 @@ private BinaryLogRecordKind Write(PropertyInitialValueSetEventArgs e)
 
         private BinaryLogRecordKind Write(EnvironmentVariableReadEventArgs e)
         {
-            WriteMessageFields(e, writeImportance: true);
+            WriteMessageFields(e, writeImportance: false);
```

Comment:
ToLearn: Why it was changed?

Reply:
The importance was never set on EnvironmentVariableReadEventArgs and was holding the default value.
In BinLog viewer it displays on the top, without respecting this property
![image](https://github.com/user-attachments/assets/e7216a2f-675a-4611-a80f-a71c4f86e927)
So I concluded it's not needed.